commit d3c91e7271a9d64d23ef5b00e868cb68bc99644b (refs/changes/85/585/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-04T00:04:32+00:00 (3 years ago)
    
    TOOLS-1558 TOOLS-1555 broke sdcnode builds: 'npm install' isn't run, buildEnv changes break on spaces (fix Makefile syntax)

diff --git a/Makefile b/Makefile
index 2c6b520..39a1b64 100644
--- a/Makefile
+++ b/Makefile
@@ -36,7 +36,7 @@ all: build/src nodes bits
 
 .PHONY: npmdeps
 npmdeps:
-    npm install
+	npm install
 
 build/src:
 	git clone https://github.com/nodejs/node.git build/src
